lib/src/binary_codec.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:convert';
 import 'dart:typed_data';
 
 import 'package:buffer/buffer.dart';
+import 'package:latlng/latlng.dart';
```

Comment:
The class used from here seems to be trivial. I'd rather have it inside `package:postgres`, as it reduces the risk that in the future it would cause conflicts. Could you please move it inside / reimplement?

Also, we could call it `PgPoint` maybe, after the postgresql type?

Reply:
Seems useful. Where should I put it?

Reply:
It could be a new `lib/model.dart` file which we export at the top-level library and use here.

Reply:
Done